data/bootstrap_hwdefinition.yaml

```diff
@@ -0,0 +1,65 @@
+---
+#############################################################################################################################
```

Comment:
Likely requires some sort of introduction on what this file contains, and a skeleton tree of the sections.  Much of the data at the top here currently seems like it could be brought inline with the fields themselves, e.g. without scrolling up, one has very little idea what the `version:` field is supposed to represent.

We should also start thinking about what fields will be required and generate an error if missing, or alternatively if not required and absent, may simply silently reduce functionality (second pass likely)